### DISCUSSION TITLE
Custom textures will not be visible after check the reset to prefab button

### DISCUSSION THREAD

**Discussion by jcx1991911** [FakeFishGames/Barotrauma#13807] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Custom textures become completely transparent (can still be selected), and this continues until the game is restarted
![20240423154042_1](https://github.com/FakeFishGames/Barotrauma/assets/116519004/86e4a57c-a00c-45f8-96ba-ecb37eb1d9ed)
![20240423154131_1](https://github.com/FakeFishGames/Barotrauma/assets/116519004/7b973173-fe0c-4620-9046-a65b6304ae79)

### Reproduction steps

check the reset to prefab button of any custom textures

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.4.3.0 (unstable)

### -

v1.4.4.1

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

I gave this a quick test, and unfortunately couldn't reproduce the issue. Tested a handful of mods I had installed, and resetting to prefab didn't break any of their textures. What exactly do you mean by "custom texture", just any new non-vanilla texture added by a mod? Would it be possible to give us some examples of mods where this occurs, or perhaps even send us the mod your working on as a .zip here if it occurs in your own mod? Thanks and sorry for the inconvenience!

**Reply by jcx1991911** [reply]
I have tested on another workshop project,the same thing happened
https://steamcommunity.com/sharedfiles/filedetails/?id=3010148225

The background of the cabin is what I call a "custom texture".
It's essentially a bundle of sub files that are released to the workshop, which is a submarine's built-in mod file.
